orchestrator/eth_event_watcher.go

```diff
@@ -82,11 +81,14 @@ func (p *peggyOrchestrator) CheckForEvents(
 		iter.Close()
 	}
 
-	log.WithFields(log.Fields{
-		"start":     startingBlock,
-		"end":       currentBlock,
-		"Contracts": erc20DeployedEvents,
-	}).Debugln("Scanned ERC20Deployed events from Ethereum")
+	p.logger.Debug().
+		Uint64("start", startingBlock).
+		Uint64("end", currentBlock).
+		// TODO: fix log. print the list of events? Or the length of it?
```

Comment:
Lets remove the TODO or create an issue for it and reference it.

Reply:
We've already discussed this. Gonna print length of lists